examples/dump_torrent.cpp

```diff
@@ -58,40 +58,37 @@ std::vector<char> load_file(std::string const& filename)
 
 int main(int argc, char* argv[]) try
 {
-	if (argc < 2 || argc > 4) {
-		std::cerr << "usage: dump_torrent torrent-file [total-items-limit] [recursion-limit]\n";
+	if (argc < 2 || argc > 5) {
+		std::cerr << "usage: dump_torrent torrent-file [total-items-limit] [recursion-limit] [piece-count-limit]\n";
 		return 1;
 	}
 
-	int item_limit = 1000000;
-	int depth_limit = 1000;
+	lt::load_torrent_limits cfg;
 
-	if (argc > 2) item_limit = atoi(argv[2]);
-	if (argc > 3) depth_limit = atoi(argv[3]);
+	if (argc > 2) cfg.max_decode_tokens = atoi(argv[2]);
+	if (argc > 3) cfg.max_decode_depth = atoi(argv[3]);
+	if (argc > 4) cfg.max_pieces = atoi(argv[4]);
 
 	std::vector<char> buf = load_file(argv[1]);
-	lt::bdecode_node e;
 	int pos = -1;
 	lt::error_code ec;
-	std::cout << "decoding. recursion limit: " << depth_limit
-		<< " total item count limit: " << item_limit << "\n";
-	int const ret = lt::bdecode(&buf[0], &buf[0] + buf.size(), e, ec, &pos
-		, depth_limit, item_limit);
+	std::cout << "decoding. recursion limit: " << cfg.max_decode_depth
```

Comment:
Shouldn't `cfg.max_pieces` also be printed here?

Reply:
well, I don't think any of these limits need to be printed. I just left these here because they were already being printed. I'm tempted to remove this print statement also, but that's a separate change.